INSTALL.md

```diff
@@ -24,7 +24,7 @@ This file is best viewed using a browser-plugin for markdown `.md` files.
   * Version 2.2 or higher with `mod_ssl` module
   * X509 host certificate.
 
-* [Database server](#rdbms) 
+* [Database server](#rdbms)
```

Comment:
This link doesn't seem to go anywhere? I expected it to take me to the instructions for setting up a database server...

Reply:
I guess it's meant to be `#database-server` in the link.

Reply:
Sure - but all I did here was remove white space!!

Reply:
```suggestion
* [Database server](#database-server)
```